src/mlbstandings/web.py

```
@@ -1,17 +1,27 @@
     @backoff.on_exception(
         backoff.expo,
-        (requests.exceptions.HTTPError, requests.exceptions.ConnectionError, requests.exceptions.Timeout),
+        (requests.exceptions.HTTPError, urllib.error.HTTPError, requests.exceptions.ConnectionError, requests.exceptions.Timeout),
         max_time=600,
         giveup=lambda e : isinstance(e, requests.exceptions.HTTPError) and e.response.status_code not in set([429, 500, 503]),
         max_value=60
     )
     def read(url: str) -> str:
         print(f'Web.read({url}')
+        if url.find('https://www.baseball-reference.com/') == 0:
+            req = urllib.request.Request(url)
```

Ruff (0.11.9)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)